tests/test_tradesimulation.py

```diff
@@ -41,7 +49,7 @@ def test_minutely_emissions_generate_performance_stats_for_last_day(self):
                                                       data_frequency='minute',
                                                       emission_rate='minute')
         algo = NoopAlgorithm(sim_params=params)
-        algo.run(source=[], overwrite_sim_params=False)
+        algo.run(data_portal=FakeDataPortal())
```

Comment:
Should we give the random data portal to this test? Or one from a trade history of 0's/NaNs?
e.g. could do.
```
        data_portal = create_data_portal(self.env,
                                         self.tempdir,
                                         params,
                                         [1])
```